winui/Cartesian-Charts/Appearance.md

```diff
@@ -167,57 +78,21 @@ this.Content = chart;
 
 {% endtabs %}
 
-![Custom palette in WinUI Chart](Appearance_images/WinUI_chart_custom_palette.png)
+![Custom PaletteBrushes in WinUI Chart](Appearance_images/WinUI_chart_custom_palette.png)
 
 ## Applying Gradient
 
```

Comment:
@YuvarajPalanisamy : Gradient working fine in cartesian chart, check with naveen and update that section.

Reply:
Added Gradient section in Appearance section